src/index.js

```diff
@@ -29,50 +22,12 @@ import save from './save';
 
 /**
  * Every block starts by registering a new block type definition.
+ * With API version 3, metadata is defined in block.json and we only
+ * provide the edit and save functions here.
  *
  * @see https://developer.wordpress.org/block-editor/developers/block-api/#registering-a-block
  */
 registerBlockType( 'create-block/meu-primeiro-block', {
-	/**
```

Comment:
This branch doesn't have an edit and save JS file. Can you keep those links as they are?